packages/livekit-rtc/src/types.ts

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2024 LiveKit, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+export interface ChatMessage {
+  id: string;
+  timestamp: number;
+  message: string;
+  editTimestamp?: number;
+  generated?: boolean;
+}
```

Comment:
as far as i understand this is the same as the proto interface but with bigints swapped out for numbers? there's already a bunch of conversions happening in code and i wonder if this cuts down on enough of them to warrant this utility type or whether it's okay to just use the proto one

Reply:
are you concerned about performance reasons of the conversions or maintenance burden?
reasons for me to keep them separate: 
- we might be adding stuff in the proto that we don't necessarily want to expose to the user facing API in the same form
- it's a bit of a pain to work with bigints in JS

Reply:
>  just a bit weird how this is the first time we've had to do this

FWIW in the client js sdk we do this for almost every proto type